Makefile

```diff
@@ -55,6 +55,15 @@ endif
 	@sed -i '' "s/\"tag\": .*/\"tag\": \"v$(VERSION)\"/" Bugsnag.podspec.json
 	@sed -i '' "s/NOTIFIER_VERSION = .*;/NOTIFIER_VERSION = @\"$(VERSION)\";/" Source/BugsnagNotifier.m
 
+# Makes a release and pushes to github/cocoapods
+release:
+	ifeq ($(VERSION),)g
+		@$(error VERSION is not defined. Run with `make VERSION=number release`)
+	endif
+		make VERSION=$(VERSION) bump && git commit -am "v$(VERSION)" && git tag v$(VERSION) \
```

Comment:
These can be on separate lines instead of `&&`-concatenating them altogether, if you find that easier to parse.